src/lib/spinel/spinel.h

```diff
@@ -718,6 +718,32 @@ enum
     SPINEL_RADIO_LINK_TREL_UDP6     = 1,
 };
 
+typedef enum
+{
+    SPINEL_SRP_CLIENT_ITEM_STATE_TO_ADD     = 0, // Item to be added/registered.
+    SPINEL_SRP_CLIENT_ITEM_STATE_ADDING     = 1, // Item is being added/registered.
+    SPINEL_SRP_CLIENT_ITEM_STATE_TO_REFRESH = 2, // Item to be refreshed (re-register to renew lease).
+    SPINEL_SRP_CLIENT_ITEM_STATE_REFRESHING = 3, // Item is being refreshed.
+    SPINEL_SRP_CLIENT_ITEM_STATE_TO_REMOVE  = 4, // Item to be removed.
+    SPINEL_SRP_CLIENT_ITEM_STATE_REMOVING   = 5, // Item is being removed.
+    SPINEL_SRP_CLIENT_ITEM_STATE_REGISTERED = 6, // Item is registered with server.
+    SPINEL_SRP_CLIENT_ITEM_STATE_REMOVED    = 7, // Item is removed.
+} spinel_srp_client_item_state_t;
+
+typedef enum
+{
+    SPINEL_SRP_CLIENT_ERROR_NONE             = 0, // No error
```

Comment:
End lines with period to be consistent with above?
```suggestion
    SPINEL_SRP_CLIENT_ERROR_NONE             = 0, // No error.
```